### DISCUSSION TITLE
Как сделать возможность добавления строк в таблицу циклов через панель?

### DISCUSSION THREAD

**Discussion by khpavel** [totumonline/totum-ru-issues-and-discussions#469] [Как это сделать?] [answered]
Если для таблицы циклов установить флажок в поле Панель, то при нажатии на кнопку Панель идет переход на первую таблицу Расчетная в цикле.
Вопрос. Как сделать возможность добавления строк в таблицу циклов через панель?

**Comment by erpic-ru** [comment, marked as answer]
https://www.youtube.com/watch?v=950C-q44REM

+ код linktopanel
